src/main/java/org/jitsi/videobridge/octo/OctoRelayService.java

```diff
@@ -87,9 +117,29 @@ public void start(BundleContext bundleContext)
     @Override
     public void stop(BundleContext bundleContext) throws Exception
     {
-        if (relay != null)
+        if (udpTransport != null)
         {
-            relay.stop();
+            udpTransport.stop();
         }
+        if (octoTransport != null)
+        {
+            octoTransport.stop();
+        }
+    }
+
+    public OctoRelayServiceStats getStats()
+    {
+        return new OctoRelayServiceStats(
```

Comment:
Can we use JSON or a map? Using that many parameters is error prone.

Reply:
I agree, and considered porting `OctoRelayService` to kotlin just to get named parameters here :) Neither JSON nor a map work great, because then the caller has to look them up by string which I wasn't crazy about.  I can change this to a builder-style so it's nice from java.

Reply:
We could have `VideobridgStatistics` export all contents of the map. Just use the existing constants so we don't break dashboards. I did something like this in jicofo recently and I think it's much cleaner.

Reply:
I tried the map approach, but it doesn't give a good way to fill default values if the relay service is null, which I think is necessary.